openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -48,13 +48,15 @@
     get_potentially_retired_user_by_username,
     get_retired_email_by_email,
     get_retired_username_by_username,
-    is_username_retired
+    is_username_retired,
+    is_email_retired
 )
 from common.djangoapps.student.models_api import (
     confirm_name_change,
     do_name_change_request,
     get_pending_name_change
 )
+from openedx.core.djangoapps.user_api import accounts
```

Comment:
where is this import used?

Reply:
L324

Reply:
I would suggest you import constant by name instead from filename. Getting data according to filename is bad practice according to Python.

Reply:
you may just import `RETIRED_EMAIL_MSG` if nothing else is being used.